### ISSUE TITLE
Build Error during npm install

### ISSUE THREAD

**Issue by sbukow** [alexa/interactive-adventure-game-tool#58]
Beginner with no experience, but having trouble during the npm install, and reading the other issues was unable to find a fix.  Error message below:

![image](https://user-images.githubusercontent.com/35157062/34646325-155bbc90-f333-11e7-9198-04f06edffb7d.png)

**Comment by mym1990** [comment]
Sbukow can you tell us what you have tried by 'reading other issues', that will help us evaluate what other possible solutions there are for you.

**Comment by sbukow** [comment]
Yes. I was referring to the other npm issues that others had posted in here   #54 #52 #42 #15, because this was not the first issue i had with the install. its just where I am at now.

I had followed the instructions #54  for install on windows and had the same errors that required the change package.json require-dir from 0.2.0 to 0.3.2.  and installed python, to take care of the python error as well. 

Following that, I am now at the error message above
